pymake/cli/main.py

```diff
@@ -25,3 +25,7 @@
   pymake.command.main(sys.argv[1:], os.environ, os.getcwd(), cb=sys.exit)
   pymake.process.ParallelContext.spin()
   assert False, "Not reached"
+
+
+if __name__ == '__main__':
+    main
```

Comment:
Hi @JeremyMarshall  -- i don't understand what problem this fixes. can you please elabourate?

this patch introduces a defect, as the new `main()` function is not actually called.